test/metabase/server/middleware/session_test.clj

```diff
@@ -395,6 +395,11 @@
 ;;; ----------------------------------------------------- Session timeout -----------------------------------------------------
 
 (deftest session-timeout-tests
+  (testing "Setting the session timeout should fail if the timeout is more than 100 years"
+    (is (thrown-with-msg?
+         java.lang.AssertionError
+         #"Session timeout must be less than 100 years"
+         (mw.session/session-timeout! {:unit "hours", :amount (* 101 365 24 60 60)}))))
```

Comment:
I think you were trying to test that whether it errors if you set it to 101 years here but `(* 101 365 24 60 60)` hours is 101 * 60 * 60 years, right?

Reply:
Yeah, thanks. Updated and added more tests  https://github.com/metabase/metabase/pull/27347/commits/f7d4b9917c7e564f21604d715c622d8fca60f17e